docs/root/intro/arch_overview/upstream/load_balancing/panic_threshold.rst

```diff
@@ -66,5 +66,23 @@ priority.
 | 5%          |  65%        |  7%      | YES          |   93%    | NO           |  98%        |
 +-------------+-------------+----------+--------------+----------+--------------+-------------+
 
+Setting the panic threshold to 0%, panic mode can be disabled.
+
+If all hosts becomes unhealthy, normalized total health is 0%, all of traffic redirect to P=0.
```

Comment:
reworded a bit:

If all hosts become unhealthy normalized total health is 0%, and if the panic threshold is above 0% all traffic will be redirected to P=0.  However, if the panic threshold is 0% for any priority, that priority will never enter panic mode.  In this case if all hosts are unhealthy, Envoy will fail to select a host and will instead immediately return error responses with "503 - no healthy upstream".